### ISSUE TITLE
Missing stashed changes

### ISSUE THREAD

**Issue by Warren-Vasavah** [desktop/desktop#17028]
### The problem

Good day! I just experience a problem after clicking the stash changes in Github Desktop after waiting about 1-2mins the changes are gone and received an error of stash failed however it didn't return the changes in my end it likes it delete the changes. I checked my recycle bin the changes are not there. I tried to retrieve the changes by using git stash pop but it only returns package.json file. There are 70-90 files changed in there, how can I retrieve it?

### Release version

Version 3.2.6 

### Operating system

Windows

**Comment by tidy-dev** [comment]
Oomph. That is a lot changes. 

Could you please upload a log file from a day you experienced the issue?
To access the log files go to the menu and select Help > Show Logs.

Just to clarify, were you attempting to pop/apply a stash or stash some changes?

**Comment by Warren-Vasavah** [comment]
Hello, thank you for your response!

I'm attempting to stash some changes using github desktop. This is how exactly happened:

Using VS code command prompt terminal. I was gonna pull a request from the master branch, however, I immediately realized that I didn't stashed my changes, so I stop request by pressing "Ctrl + C" then typed "y" then enter. Then I go to github desktop and stash the changes. I noticed that it didn't stashed the changes faster how it should be then I clicked again the stash changes then waited about 1-2mins then it was gone. I checked the terminal in VS code it pops a message "stash failed". Then I searched for a way to retrieve, and I used git stash pop. But it returned only the package.json file

I'm uploading the log files. I think it happened 2023-07-05. 
[2023-07-05.desktop.production.log](https://github.com/desktop/desktop/files/11987125/2023-07-05.desktop.production.log)

But I will send also the logs for July 4 and July 6 for more info
[2023-07-03.desktop.production.log](https://github.com/desktop/desktop/files/11987135/2023-07-03.desktop.production.log)
[2023-07-04.desktop.production.log](https://github.com/desktop/desktop/files/11987136/2023-07-04.desktop.production.log)
[2023-07-06.desktop.production.log](https://github.com/desktop/desktop/files/11987138/2023-07-06.desktop.production.log)

**Comment by Warren-Vasavah** [comment]
Hello, thank you for your response!

I'm attempting to stash some changes using github desktop. This is how
exactly happened:

Using VS code command prompt terminal. I was gonna pull a request from the
master branch, however, I immediately realized that I didn't stashed my
changes, so I stop request by pressing "Ctrl + C" the moment I realized.
Then I go to github desktop and stash the changes. I noticed that it didn't
stashed the changes faster how it should be then I clicked again the stash
changes then waited about 1-2mins then it was gone. I checked the terminal
in VS code it pops a message "stash failed". Then I searched for a way to
retrieve, and I used git stash

I'm uploading the log files. I think it happened 2023-07-05.


But I will also send the logs for July 4 and July 6 for more information.

I also replied directly on Github. Thank you
